scripts/run.sh

```diff
@@ -72,15 +72,15 @@ elif [ $1 == "test" ]; then
         echo
 
         if [[ $backend_rc == 0 && $frontend_rc == 0 ]]; then
-            echo "All tests pass!!!"
-            if [[ $GITHUB_HEAD_REPO_URL == "https://github.com/mozilla/balrog.git" ]];
+            if [ '$GITHUB_HEAD_REPO_URL' == "https://github.com/mozilla/balrog.git" ];
```

Comment:
If we set a new variable on taskcluster that the condition can be based on, i think that should work. What do you think @mozbhearsum ?